README.rst

```diff
@@ -234,6 +234,9 @@ it should be easier to use in most cases.
     response.status is also set to HTTP error code.
 
   This option is set to True by default if you use SplashRequest.
+  Note that if you are using ``render.html`` endpoint, only response.url
```

Comment:
Oh, gotcha! So we can either document this right here, or just state that not all keys are present in all endpoints at the start. The first option feels more user-friendly, if this is not going to change in splash soon.